pkg/runner/runner_test.go

```diff
@@ -105,6 +105,7 @@ func TestRunEvent(t *testing.T) {
 		{"testdata", "defaults-run", "push", "", platforms, ""},
 		{"testdata", "uses-composite", "push", "", platforms, ""},
 		{"testdata", "issue-597", "push", "", platforms, ""},
+		{"testdata", "env-and-path", "push", "", platforms, "linux/arm64"},
```

Comment:
since tests were updated to not include arch
```suggestion
		{"testdata", "env-and-path", "push", "", platforms, ""},
```

Reply:
Fixed it @catthehacker